code/modules/hydroponics/plant_genes.dm

```diff
@@ -212,7 +212,7 @@
 	// Cell recharging trait. Charges all mob's power cells to (potency*rate)% mark when eaten.
 	// Generates sparks on squash.
 	// Small (potency*rate*5) chance to shock squish or slip target for (potency*rate*5) damage.
-	// Multiplies max charge by (rate*1000) when used in potato power cells.
+	// Also affects plant batteries see capatative cell production datum
```

Comment:
@ChangelingRain placed this comment here instead of a copypaste of the one that is at line 358